svelte.config.cjs

```diff
@@ -0,0 +1,25 @@
+const sveltePreprocess = require('svelte-preprocess');
+const node = require('@sveltejs/adapter-node');
+const pkg = require('./package.json');
+
+/** @type {import('@sveltejs/kit').Config} */
+module.exports = {
+	// Consult https://github.com/sveltejs/svelte-preprocess
+	// for more information about preprocessors
+	preprocess: sveltePreprocess(),
+	kit: {
+		// By default, `npm run build` will create a standard Node app.
+		// You can create optimized builds for different platforms by
+		// specifying a different adapter
+		adapter: node(),
```

Comment:
Please use adapter: static(), if possible (as I commented in previous pull request).

Please keep the pull request open (no need to close pull request while resolving code reviews and create new one - just keep adding new commits please). We will close pull request once it is merged.

Reply:
I am not added to this repo so I forked this repo that is why I can not commit this repo.